11.2.основы_работы_со_списками/11.2.9.py

```diff
@@ -0,0 +1,5 @@
+numbers1 = [1, 2, 3]
+numbers2 = [6] * 9
+numbers3 = [7, 8, 9, 10, 11, 12, 13]
+result = numbers1 + numbers1 + numbers2 + numbers3
+print(result)
```

Comment:
Так странно, `numbers2` нужно повторить 9 раз - ты умножил, `numbers1` нужно повторить 2 раза - ты сложил xD

Лучше сложение заменить произведением:
```suggestion
numbers1 = [1, 2, 3] * 2
numbers2 = [6] * 9
numbers3 = [7, 8, 9, 10, 11, 12, 13]
result = numbers1 + numbers2 + numbers3
print(result)
```